teaser/project.py

```diff
@@ -80,9 +78,9 @@ def __init__(self, load_data=True):
         self.modelica_info = ModelicaInfo()
 
         self.modelica_project = self.name
-        self.weather_file_header = "wetter"
+        # self.weather_file_header = "wetter"
         self.weather_file_path = "modelica://AixLib/Resources/WeatherData/" \
-                                 "TRY2010_12_Jahr_Modelica-Library.txt"
+                                 "USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.mos"
```

Comment:
Is this the weather file we want to use as default? given that all other TEASER modules rely on german data?